docs/pages/server-access/troubleshooting-server.mdx

```diff
@@ -6,7 +6,27 @@ description: Describes common issues and solutions for access to servers.
 This section describes common issues that you might encounter in managing access to servers
 with Teleport and how to work around or resolve them.
 
-## Starting SSH sessions fails
+## Starting SSH sessions fails with expired certificates
+
+A ssh session fails to start on certain SSH servers with the error message
+of expired certificates. If a server's date and time is off enough that can
+result in not accepting certificates.
+
+### Symptom
+
+A user attempts to connect to a machine and can receive a message including the following.
+
+```text
+[ssh: cert has expired]
+```
+
+### Solution
+
+Check the system date and time on the target machine (ex: `date`). Often the 
```

Comment:
You need to check both the auth server and the target machine. The target machine can be correct but if the auth server's clock is incorrect it won't matter.

Reply:
Thanks. Will have to think of way to say it that works with Teleport Cloud too.